engine/pom.xml

```diff
@@ -28,10 +28,38 @@
     <artifactId>engine</artifactId>
     <properties>
         <spark.version>1.3.1</spark.version>
-        <drools-version>6.2.0.Final</drools-version>
+        <drools-version>6.3.0.Final</drools-version>
         <outputFileName>stratio-decision</outputFileName>
     </properties>
+
+    <repositories>
+        <repository>
+            <id>cafeina</id>
+            <url>http://10.200.0.96:8080/kie-drools-wb-distribution-wars-6.3.0.Final-wildfly8/maven2</url>
```

Comment:
It would be possible to have this repository in user settings.xml, in order to avoid open source product user who tried to compile Decision get errors